core/server/data/schema/schema.js

```diff
@@ -300,5 +300,34 @@ module.exports = {
         created_by: {type: 'string', maxlength: 24, nullable: false},
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    integrations: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        name: {type: 'string', maxlength: 191, nullable: false},
+        slug: {type: 'string', maxlength: 191, nullable: false, unique: true},
+        icon: {type: 'string', maxlength: 2000, nullable: true},
+        description: {type: 'string', maxlength: 2000, nullable: true},
+        created_at: {type: 'dateTime', nullable: false},
+        created_by: {type: 'string', maxlength: 24, nullable: false},
+        updated_at: {type: 'dateTime', nullable: true},
+        updated_by: {type: 'string', maxlength: 24, nullable: true}
+    },
+    api_keys: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        type: {
+            type: 'string',
+            maxlength: 50,
```

Comment:
could be a lot smaller?

Reply:
True. I was following the size we use for all our other `type` columns, in the past we've typically erred on the side of larger columns than we initially expect we need. I'm not sure of the exact reason, maybe @kirrg001 can enlighten us 😄

I vaguely remember issues with db encoding which meant that the length should always be twice the largest value we expect - I could be well off there though

Reply:
Yeah fair enough - not really a big deal tbh! Would be interested to hear the reasoning behind that though :dancer:

Reply:
We currently follow these definitions: https://github.com/TryGhost/Ghost/pull/7932